src/dbnode/config/m3dbnode-local-etcd.yml

```diff
@@ -98,3 +98,9 @@ db:
           initialCluster:
               - hostID: m3db_local
                 endpoint: http://127.0.0.1:2380
+
+  # Uncomment this to enable local jaeger tracing. See https://www.jaegertracing.io/docs/1.9/getting-started/
+  # for quick local setup (which this config will send data to).
+
+  tracing:
```

Comment:
hm, should this be commented out?

Reply:
also, should it be in all the config files?

Reply:
Yes - but instead of this, I'd rather just have an annotated config section like http://m3db.github.io/m3/query_engine/config/annotated_config.yaml. I can add them to all of the configs but I think that's overkill.